target_snowflake/db_sync.py

```diff
@@ -144,15 +144,25 @@ def flatten_schema(d, parent_key=[], sep='__', level=0, max_level=0):
 
     return dict(sorted_items)
 
+def _should_json_dump_value(key, value, flatten_schema=None):
+    if type(value) is list or type(value) is dict:
```

Comment:
pylint prefers `isinstance()` for typechecks:
`# causes C0123 Using type() instead of isinstance() for a typecheck`

https://github.com/codacy/codacy-pylint/blob/master/docs/description/C0123.md